board/linux/libiosyscalls/src/iosyscalls.cpp

```diff
@@ -23,13 +22,15 @@ namespace
     constexpr auto ENV_NAME       = "IOSYSCALLS_REDIRECT_TO_IMAGE";
     constexpr auto FIRST_FILEDESC = 64'566'756;
     constexpr auto SYSROOT        = "sysroot";
+    constexpr auto SYSTEM_PARTITION = "system_a";
+    constexpr auto SYSTEM_DATA_DIR  = "data";
     bool g_evaluated              = false;
     bool g_redirect               = false;
 
     constexpr const char *LINUX_PATHS[]{
         "/dev/", "/etc/", "/lib", "/usr/share", "/run/user", "/home", "/proc", "/tmp", "MuditaOS.log", nullptr};
 
-    constexpr const char *IMAGE_PATHS[]{"/sys", "/mfgconf", "sys", "assets", "country-codes.db", "Luts.bin", nullptr};
+    constexpr const char *IMAGE_PATHS[]{"/system", "/mfgconf", "/user", nullptr};
```

Comment:
What do you think about using initializer lists instead of arrays ending with `nullptr`?
```
const auto IMAGE_PATHS = {"/system", "/mfgconf", "/user"};
for (const auto path : IMAGE_PATHS);
```

Reply:
As you wish - fixed! :P